.github/workflows/publish-latest-react-v4.yml

```diff
@@ -0,0 +1,119 @@
+# Description: this workflow publishes UI packages to `latest` tag on NPM.
```

Comment:
If you are copying these workflows from main, do not forget to remove the metrics reporting section! A failure in Test / Publish latest triggers a sev2.